docs/api/create-list.md

```diff
@@ -1,66 +1,73 @@
 <!--[meta]
 section: api
-title: createList
+title: Create List
+order: 2
 [meta]-->
 
-# keystone.createList(name, options)
+# createList(name, options)
```

Comment:
I'd like to keep the `keystone.` part on this heading so it's obvious what method we're talking about (as it's the main page heading).

```suggestion
# `keystone.createList(name, options)`
```

Reply:
This is why I've chose to follow a format where usage always follows the main heading